examples/player/simid_player.html

```diff
@@ -50,11 +50,49 @@
       simidPlayer.initializeAd();
     }
 
+    /* Disables or enables stop, skip, and fatal error buttons
```

Comment:
make this a proper js doc
```
/**
 * Description of the method.
 * @param {paramType} paramName Description of the param.
 */
```

Reply:
wasn't sure if I should do that in html file, but will change now